### PR TITLE
[FIX] product image import

### PR DESCRIPTION
**Pull request by AurelijaNorvaisaite**
https://hive.versada.eu/projects/metalurga-upgrade/work_packages/22467/activity

### CODE REVIEW

File: connector_prestashop/models/product_image/importer.py [hunk 1]
```diff
@@ -76,6 +76,6 @@ def run(self, template_id, image_id, **kwargs):
         # )
         if str(product_tmpl.default_image_id) != str(image_id):
             return
-        self.binder_for("prestashop.product.image")
+        binder = self.binder_for("prestashop.product.image")
```
Comment (paulius-sladkevicius) [inline connector_prestashop/models/product_image/importer.py:79]: How obvious error, I suppose it was in the logs too :-D I couldn't see it..

**Review by paulius-sladkevicius — APPROVED** [review]

**Comment by paulius-sladkevicius** [comment]
no need for an extra review

File: connector_prestashop/models/product_image/importer.py (hunk 1 above)
Comment (paulius-sladkevicius) [reply connector_prestashop/models/product_image/importer.py:79]: oh, a `binder` was declared above.

Comment (AurelijaNorvaisaite) [reply connector_prestashop/models/product_image/importer.py:79]: Yeah, it took some time to understand what's going on as it looked the same in previous versions too.

**Pull request merged**